yarn.lock

```diff
@@ -1976,10 +1976,26 @@
     "@types/scheduler" "*"
     csstype "^3.0.2"
 
+<<<<<<< HEAD
```

Comment:
Your yarn.lock file is invalid, this probably causes wrong versions of things to be installed, looks like a merge gone wrong